owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/SecurityUtils.kt

```diff
@@ -0,0 +1,39 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Juan Carlos Garrote Gascón
+ *
+ * Copyright (C) 2021 ownCloud GmbH.
+ * <p>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ * <p>
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * <p>
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.presentation.ui.security
+
+const val LOCK_TIMEOUT = "lock_timeout"
+const val LAST_UNLOCK_TIMESTAMP = "last_unlock_timestamp"
+
+enum class LockTimeout {
+    IMMEDIATELY, ONE_MINUTE, FIVE_MINUTES, THIRTY_MINUTES;
+
+    companion object {
+        fun fromStringToMilliseconds(string: String?): Int {
+            return when {
+                string.equals("IMMEDIATELY", ignoreCase = true) -> 0
+                string.equals("ONE_MINUTE", ignoreCase = true) -> 60000
+                string.equals("FIVE_MINUTES", ignoreCase = true) -> 300000
+                else -> 1800000
```

Comment:
Several things here:
1. Let's use [`_` to write big numbers:](https://kotlinlang.org/docs/basic-types.html#literal-constants) 300_000, 60_000 etc. It's easier to read 👍  
2. In case the `string` is null or a not expected value, the app will be unprotected for 30 minutes, right? Should this be the expected behavior?
3. This function takes care of checking the LockTimeout, and then transform it to millis. The first part, is basically the enum function `valueOf()` right? [Let's check the docs about enums in kotlin](https://kotlinlang.org/docs/enum-classes.html#working-with-enum-constants). Maybe we could write this in a different and better way 👍

Reply:
Ok, I changed the logic following your advice, you tell me 👍